app/step_planners/step_planner/move_barcodes_from_tube_rack_to_plate.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
```

Comment:
This file was previously a 'script' and involved spinning up an entire separate process. It was mostly encapsulated in a class already, so I moved it into this file.  Same for the next file.